crates/matrix-sdk-ui/tests/integration/timeline/pinned_event.rs

```diff
@@ -132,7 +139,10 @@ async fn test_new_pinned_event_ids_reload_the_timeline() {
 
     let room = test_helper.client.get_room(&room_id).unwrap();
     let timeline = Timeline::builder(&room)
-        .with_focus(TimelineFocus::PinnedEvents { max_events_to_load: 100 })
+        .with_focus(TimelineFocus::PinnedEvents {
+            max_events_to_load: 100,
+            max_concurrent_requests: 10,
+        })
```

Comment:
Can you make a tiny test helper that creates a pinned events focused timeline, instead? that will avoid repeating the new argument everywhere